Amazon.IonObjectMapper.Test/Utils.cs

```diff
@@ -123,7 +123,7 @@ public static T DeserializeWithCustomSerializer<T>(IonSerializer<T> customSerial
         public static T DeserializeWithCustomSerializers<T>(Dictionary<Type, IIonSerializer> ionSerializers, T item)
         {
             var serializer = new IonSerializer(new IonSerializationOptions { IonSerializers = ionSerializers });
-
+            
```

Comment:
Nit: let's undo this spacing change since nothing is happening to this file.

Reply:
Reverted.